### PR TITLE
check untyped defs

### PR DESCRIPTION
**Pull request by kitagry**
strict check for mypy

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -66,6 +66,8 @@ quote-style = "single"
 ignore_missing_imports = true
 plugins = ["gokart.mypy:plugin"]
 
+check_untyped_defs = true
```
Comment (kitagry) [inline pyproject.toml:69]: Turn on `check_untyped_defs` option.

**Review by hiro-o918 — APPROVED** [review]
LGTM

**Review by hirosassa — APPROVED** [review]
Looks good👍

**Review by yokomotod — APPROVED** [review]
Nice job ! towards `strict = true` ...

**Pull request merged**